src/connectors/siriusxm-player.ts

```diff
@@ -15,20 +15,18 @@ const filter = MetadataFilter.createFilter({
 	track: [removeYear, removeCover, removeExclusive],
 });
 
-Connector.playerSelector = '.sxm-player-controls';
-Connector.artistSelector = '.sxm-player-controls .artist-name';
-Connector.trackSelector = '.sxm-player-controls .track-name';
+Connector.playerSelector = '._playbackControls_1cag9_17';
```

Comment:
While this will work - it feels like the connector is going to be super brittle to any new publish of the site as I suspect these IDs will mutate. Is there any other DOM selectors?

Reply:
I agree. The new site smells of auto generated classes from a framework. There aren't good classes or IDs to use.

Another approach would be to try selectors that match the base of the class without the IDs. I haven't tried that before and I'm open to suggestions.

Reply:
I'm unable to use the site to see what the DOM is like due to georestrictions so I suspect this is better than nothing. happy to accept it in its current form since the site is currently broken.

Reply:
Thanks for updating to support the re-design!

To chime in here, I dealt with a similar selector creating some custom css for the new re-design - https://userstyles.world/style/13696/siriusxm-com-dec-2023

Are you able to use wildcard attribute selector, like
```
*[class^='_playbackControls']
```

Would web-scrobbler support the attribute selector syntax?

Reply:
I appreciate the suggestion and have implemented it in #4407